### DISCUSSION TITLE
Querying a datetime that is 0000-00-00 00:00:00

### DISCUSSION THREAD

**Discussion by binary64** [prisma/prisma#6105] [Q&A] [answered]
When I have 0000-00-00 00:00:00 as a datetime in my mysql db, and query it in prisma, I get error P2020:

```{
  "errors": [
    {
      "message": "\nInvalid `prisma.apiDebug.findUnique()` invocation:\n\n\n  Value out of range for the type. The column `created_date` contained an invalid datetime value with either day or month set to zero.",
      "locations": [
        {
          "line": 2,
          "column": 3
        }
      ],
      "path": [
        "apiDebug"
      ],
      "extensions": {
        "code": "INTERNAL_SERVER_ERROR",
        "exception": {
          "code": "P2020",
          "clientVersion": "2.17.0",
          "meta": {
```

How can I return null instead, for this special date?

**Comment by janpio** [comment, marked as answer]
There is an open issue for this strange MySQL special date: https://github.com/prisma/prisma/issues/5006

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
